assets/src/edit-story/components/form/dateTime/timeZone.js

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { getSettings, format } from '@web-stories-wp/date';
+import getTimeZoneString from '@web-stories-wp/date/src/getTimeZoneString';
```

Comment:
This is not a default export, but it's available via the `getOptions()` function, which is exported.

Let's either properly export `getTimeZoneString` in `packages/date/src/index.js`, or import `getOptions` here instead.